client/components/resource-details/ResourceInfo.tsx

```diff
@@ -2,7 +2,7 @@ import { useRouter } from 'next/router'
 import styled from 'styled-components'
 import { colours, fontWeight, PageTitle, largerThan, smallerThan, width } from '../../styles'
 import { Resource } from '../../utils'
-import { Review, Faculty } from '../../utils/types'
+import { Review, Faculty, Status } from '../../utils/types'
```

Comment:
unused imports, I think

```suggestion
import { Review } from '../../utils/types'
```